language/stdlib/modules/registered_currencies.move

```diff
@@ -0,0 +1,40 @@
+address 0x0:
+
+module RegisteredCurrencies {
+    use 0x0::Vector;
+    use 0x0::Transaction;
+    use 0x0::LibraConfig;
+
+    // An on-chain config holding all of the ticker symbols for registered
+    // currencies. Must be named "T" for an on-chain config.
+    struct T {
+        tickers: vector<vector<u8>>,
```

Comment:
out of curiosity why aren't we calling tickers currencies?

Reply:
Good question! This is because these are the serialized version of the ticker symbols e.g. "LBR". Currencies I worry is less explicit.

That being said, after doing some googling, I'm actually thinking "code" is a better name for this since these are technically not "ticker symbols" so much as "currency codes". I will updated :)

Reply:
@phoenix-antigravity will be sad. He really liked the naming here.